profiles/schema.py

```diff
@@ -0,0 +1,118 @@
+import graphene
+from django.conf import settings
+from graphene_django.types import DjangoObjectType
+from graphql_jwt.decorators import login_required
+from munigeo.models import AdministrativeDivision
+from thesaurus.models import Concept
+
+from .models import Profile
+
+
+class ConceptType(DjangoObjectType):
+    class Meta:
+        model = Concept
+        fields = ("code",)
+
+    vocabulary = graphene.String()
+    label = graphene.String()
+
+    def resolve_vocabulary(self, info, **kwargs):
+        return self.vocabulary.prefix
+
+
+class AdministrativeDivisionType(DjangoObjectType):
+    class Meta:
+        model = AdministrativeDivision
+        fields = ("type", "children", "origin_id", "ocd_id", "municipality")
```

Comment:
could you please migrate [this logic](https://github.com/City-of-Helsinki/open-city-profile/blob/develop/profiles/api.py#L140) to `resolve_children` in this class

Reply:
essentially, you just need to use this filter there:
```python
children = obj.children.filter(type__type="sub_district")
```

Reply:
also, type will need its own resolver as well, since in the rest code it is a [slug field](https://github.com/City-of-Helsinki/open-city-profile/blob/develop/profiles/api.py#L126)